libs/blocks/src/lib/navigation/desktop/desktop.nav-item.tsx

```diff
@@ -1,21 +1,31 @@
+import React, { useEffect, useState } from 'react';
 import { CustomLink } from '@deriv-com/components';
 import { useLanguageSwitcher, useNavigation } from '@deriv-com/hooks';
 import { NavItem } from '@deriv-com/providers';
 import { Text } from '@deriv/quill-design';
-import React from 'react';
-
 export interface DesktopNavItemProps {
   item: NavItem;
   navItemName: string;
 }
-
+const removeTrailingSlash = (str: string) => {
+  return str.replace(/\/+$/, '');
+};
 export const DesktopNavItem: React.FC<DesktopNavItemProps> = ({
   item,
   navItemName,
 }) => {
   const { onListItemHover } = useNavigation();
   const { setShowLangContent } = useLanguageSwitcher();
-
+  const [isActive, setIsActive] = useState(false);
+  useEffect(() => {
+    if (item.type === 'direct') {
+      const { pathname } = window.location;
```

Comment:
if `window` is not defined `pathname` will throw an error here. Can we use  this check? 
```
typeof window !== 'undefined'
```